olm/quickstarts/rhosak-openshift-quarkus-bind-quickstart.yaml

```diff
@@ -3,223 +3,230 @@ kind: ConsoleQuickStart
 metadata:
   name: rhosak-devsandbox-quarkus-bind-cli-toolscontainer-quickstart
 spec:
-  displayName: Binding your Quarkus application to Streams for Apache Kafka
+  displayName: Binding an application to Red Hat OpenShift Streams for Apache Kafka using the CLI
   tags:
     - streams
     - kafka
     - quarkus
   durationMinutes: 10
-  description: Binding your Quarkus application to the OpenShift Streams for Apache Kafka cloud service
+  description: Binding an application in OpenShift to Red Hat OpenShift Streams for Apache Kafka using the CLI
   prerequisites:
-    - Access to Red Hat OpenShift Streams for Apache Kafka (for more information, visit https://cloud.redhat.com/application-services).
-    - A running OpenShift Streams for Apache Kafka instance.
-    - A connection (KafkaConnection) betweeen your OpenShift project and your OpenShift Streams for Apache Kafka instance.
-  introduction: >-
-    ### This quick start shows you how to bind your Quarkus application to your Red Hat OpenShift Streams for Apache Kafka instance.
-
-    In this Quick Start, you'll bind a Quarkus application to your Kafka instance using Kubernetes Service Binding.
-    Service Binding allows you to communicate connection details and secrets to an application to allow it to bind to a service.
-    In this context, a service can be anything: a Kafka instance, a NoSQL database, etc.
-
-    By using Service Binding, we no longer need to configure connection details (host, port) authentication mechanisms (SASL, OAuth) and
-    credentials (username/password, client id/client secret) in an application. Instead, Service Binding injects these variables into your
-    application container (as files or environment variables), for your application to consume.
-
-    The Quarkus Kubernetes Service Binding extension enables Quarkus applications to automatically pickup these variables, injected as files, from the container's
-    filesystem, removing the need to specify any configuration settings in the application resources (e.g configuration files) themselves.
+    - You've completed the **Connecting OpenShift to Streams for Apache Kafka using the CLI** quick start.
+    - You have privileges to deploy applications in the OpenShift project that you connected your Kafka instance to.
+    - Your Kafka instance in [Streams for Apache Kafka](https://cloud.redhat.com/beta/application-services/streams/) is in the **Ready** state.
+
+  introduction: |-
+      ### This quick start shows how to use the Red Hat OpenShift Application Services (RHOAS) CLI to bind an example Quarkus application to a Kafka instance in Red Hat OpenShift Streams for Apache Kafka.
+
+      As a developer of applications and services, you can connect applications running on OpenShift to Kafka instances in Streams for Apache Kafka. Using an Operator to provide an application on OpenShift with the parameters required to connect to a Kafka instance is called *service binding*.
```

Comment:
I don't think this sentence is correct: "Using an Operator to provide an application on OpenShift with the parameters required to connect to a Kafka instance is called *service binding*."

We call it "Service Binding" because we use the "Service Binding Operator" to bind the connection details from our KafkaConnection to the application. I'm not really sure how to properly reword this though ...

Reply:
ACK, @DuncanDoyle. In the service binding guide, I updated this as follows:

"In {product}, you can use a specialized Operator called the Service Binding Operator to automatically provide an application on Kubernetes with the parameters required to connect to a Kafka instance. This process is called __service binding__."

Reply:
Yes, correct.